tower/src/util/combinators/map_ok.rs

```diff
@@ -0,0 +1,64 @@
+use futures_util::{future::MapOk as MapOkFut, TryFutureExt};
+use std::marker::PhantomData;
+use std::task::{Context, Poll};
+use tower_layer::Layer;
+use tower_service::Service;
+
+#[derive(Debug)]
+pub struct MapOk<S, F> {
```

Comment:
Similarly, let's add some quick doc comments here so that CI stops whining about it.